nipy/algorithms/statistics/utils.py

```diff
@@ -38,37 +38,21 @@ def multiple_fast_inv(a):
     This function is borrowed from scipy.linalg.inv, 
     but with some customizations for speed-up.
     """
+    # from scipy.linalg.decomp import _asarray_validated
+    from scipy.linalg.lapack import get_lapack_funcs
     if a.shape[1] != a.shape[2]:
         raise ValueError('a must have shape(n_samples, n_dim, n_dim)')
-    from scipy.linalg import calc_lwork
-    from scipy.linalg.lapack import  get_lapack_funcs
-    a1, n = a[0], a.shape[0]
-    getrf, getri = get_lapack_funcs(('getrf', 'getri'), (a1,))
-    for i in range(n):
-        if (getrf.module_name[:7] == 'clapack'
-            and getri.module_name[:7] != 'clapack'):
-            # ATLAS 3.2.1 has getrf but not getri.
```

Comment:
I don't really understand the `clapack` `flapack` thing.  Does it matter which one we find?  I notice that the Debian builds I have fine `flapack` versions of these for scipy back to 0.7.0.